years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+if (grep -r "Copyright[[:space:]]\+(c)[[:space:]]\+2009-.*rultor.com" --exclude-dir "target" --exclude-dir ".git" --exclude "years.sh" . | grep -v 2009-`date +%Y`) then
+    echo "Files above have wrong years in copyrights"
```

Comment:
@vkuchyn can you somehow reduce line length to 80 characters? It's hard to review now